sdks/java/maven-archetypes/examples/src/main/resources/archetype-resources/pom.xml

```diff
@@ -101,6 +101,17 @@
         </plugin>
       </plugins>
     </pluginManagement>
+
+    <extensions>
+      <!-- Use os-maven-plugin to initialize the "os.detected" properties for Bigtable
```

Comment:
That's true, but unfortunately the version of gRPC that works with Bigtable does not support the uber jar. Soon this will all be a thing of the past but we have this unfortunate stop-gap for now.